canopen_402/include/canopen_402/motor.h

```diff
@@ -134,7 +134,7 @@ class Mode {
     virtual bool start() = 0;
     virtual bool read(const uint16_t &sw) = 0;
     virtual bool write(OpModeAccesser& cw) = 0;
-    virtual bool setTarget(const double &val) { LOG("not implemented"); return false; }
+    virtual bool setTarget(const double &val) { ROSCANOPEN_ERROR("canopen_402", "Mode::setTarget implemented"); return false; }
```

Comment:
Left out "not" intentionally?
```suggestion
    virtual bool setTarget(const double &val) { ROSCANOPEN_ERROR("canopen_402", "Mode::setTarget not implemented"); return false; }
```

Reply:
Ehm, no ;)